### PR TITLE
Issue #1656

### PR DESCRIPTION
**Pull request by calvinchengx**

### CODE REVIEW

File: context/input.go [hunk 1]
```diff
@@ -326,9 +334,18 @@ func (input *BeegoInput) Session(key interface{}) interface{} {
 
 // CopyBody returns the raw request body data as bytes.
 func (input *BeegoInput) CopyBody(MaxMemory int64) []byte {
-	safe := &io.LimitedReader{R: input.Context.Request.Body, N: MaxMemory}
+	var readable reader
```
Comment (youngsterxyf) [inline context/input.go]: not conciseness enough, maybe

**Comment by JessonChan** [comment]
the code smells not so good

File: context/input.go (hunk 1 above)
Comment (youngsterxyf) [inline context/input.go]: and i think that NOT fixing this issue may be better.

**Comment by JessonChan** [comment]
@calvinchengx  you should send a pr to develop branch not master

**Pull request closed without merging**